WasabiWords.lua

```diff
@@ -609,7 +664,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int("MPX_SALV23_GEN_BS", -1) -- polgauntlet trade price
         stats.set_int("MPX_SALV23_SCOPE_BS", -1) -- police5 trade price
         stats.set_int("MPX_MOST_TIME_ON_3_PLUS_STARS", 300000) -- police4 trade price
-        stats.set_int("MPX_LOWRIDER_FLOW_COMPLETE", 1)
+        stats.set_int('MPX_LOWRIDER_FLOW_COMPLETE', 3)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_PROG', 9)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_CALL', 9)
```

Comment:
These control the user's progress through the low rider missions. Setting them to 9 will keep the user forever stuck at the end. Remove this.

Reply:
Thanks for the info, I remove it.